pkg/status/render.go

```diff
@@ -137,6 +139,17 @@ func FormatSecurityAgentStatus(data []byte) (string, error) {
 	return b.String(), nil
 }
 
+// FormatProcessAgentStatus takes a json bytestring and prints out the formatted status for process-agent
+func FormatProcessAgentStatus(data []byte) (string, error) {
+	var b = new(bytes.Buffer)
+
+	stats := make(map[string]interface{})
+	json.Unmarshal(data, &stats) //nolint:errcheck
+	renderStatusTemplate(b, "/process-agent.tmpl", stats)
```

Comment:
Do we know that the path for this will still be valid after we build and install the agent?

Reply:
Other agents are using this same pattern to render the status template e.g. security-agent:
https://github.com/DataDog/datadog-agent/blob/105db35920997fbf12e99ea24ca25096ca08eb29/pkg/status/render.go#L134

Since the files are located in the `templates` folder in the `status` package, I believe that they're compiled with the agent so they're always available.

Reply:
Awesome! I figured that was the case, I just wanted to make sure.